src/engine/math_util.c

```diff
@@ -912,17 +912,17 @@ void vec3f_get_dist_and_lateral_dist_and_angle(Vec3f from, Vec3f to, f32 *dist,
     to[1] = (from[1] + (dist * sins(pitch))); \
     to[2] = (from[2] + (dcos * coss(yaw  ))); \
 }
-void vec3f_set_dist_and_angle(Vec3f from, Vec3f to, f32 dist, Angle32 pitch, Angle32 yaw) {
+void vec3f_set_dist_and_angle(Vec3f from, Vec3f to, f32 dist, Angle pitch, Angle yaw) {
```

Comment:
whats the point of these angle types if the repo almost never uses them? i think it would also be best to remove these typedefs and replace with s16 so people don't mistakenly think there's support for changing angle types

Reply:
Yeah I think an overabundance of typedefs is a readability downgrade if anything.

Reply:
Reverted all of these to s16